index.html

```diff
@@ -234,29 +234,26 @@ <h2 class="foot-section-h" id="contact">Get started</h2>
         
         <p class="foot-section-p">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
     
-        <form action="https://formspree.io/f/xaykaovn" method="post" >
+        <form action="https://formspree.io/f/xaykaovn" method="post" id="form" >
           
             <div class="name-email-divs">
                 <div class="your-name-container">                        
-                     <!-- <div>Your name</div> -->
                      <label for="text">Your name</label>
-                    <input type="text" name="your name" placeholder="Anastasiia" maxlength='30' required id="text">
+                    <input type="text" name="your name" placeholder="Anastasiia" maxlength='30'  id="names">
                 </div>              
             
                 <div class="email-container">
-                    <!-- <div>Your email</div> -->
                     <label for="email">Your email</label>
-                    <input type="email" placeholder="|"  name="your email" id="email" required >
+                    <input type="email" placeholder="|"   name="your email" id="email" >
```

Comment:
Great job @Eldonratzlaff 🤝

- [ ] You have done a great job. However, the form is not showing the error for uppercase letters in the email, neither is the form being submitted to formspree for lowercase emails.
![form-error](https://user-images.githubusercontent.com/63917725/211171087-1d4d0d0e-6524-4e3d-9307-87d2afc22688.PNG)